server/tsd.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": "v4",
+  "repo": "borisyankov/DefinitelyTyped",
```

Comment:
Nope. I think it represent the source of the files. But 14 days ago, in the repo ```DefinitelyTyped/DefinitelyTyped``` all those links seems to have changed to ```DefinitelyTyped/DefinitelyTyped```